src/components/Main/Category/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import Image from 'next/image';
+import React from 'react';
+
+export interface ItemProps {
+  height?: number;
+}
+
+export interface CategoryProps extends ItemProps {
+  title: string;
+  contants: {
+    textFirst: string;
+    textSecond?: string;
+    image: any;
```

Comment:
```suggestion
    image: SVGElement;
```
svg 받아오는거면 SVGElement 타입이 있습니다 !!

Reply:
> svg 받아오는거면 SVGElement 타입이 있습니다 !!

SVGElement를 넣어보니 오류가 떠서요...! 더 찾아보니 string 넣으면 돼서 해결했습니다 :)